### PR TITLE
fix: adjust asset endpoint

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

**Review by tewshi — APPROVED** [review]
lgtm!

File: frontend/app/src/components/asset-manager/ManagedAssetForm.vue [hunk 1]
```diff
@@ -222,6 +218,9 @@ const saveAsset = async () => {
 
 const getUnderlyingTokenErrors = (underlyingTokens: any) => {
```
Comment (kelsos) [inline frontend/app/src/components/asset-manager/ManagedAssetForm.vue:219]: not related to the PR but kind of know the proper formats of the underlying token errors based on the checks so maybe we can replace any?

File: frontend/app/src/components/asset-manager/ManagedAssetForm.vue [hunk 2]
```diff
@@ -235,44 +234,40 @@ const getUnderlyingTokenErrors = (underlyingTokens: any) => {
 };
 
 const handleError = (message: any) => {
```
Comment (kelsos) [inline frontend/app/src/components/asset-manager/ManagedAssetForm.vue:236]: Also here maybe properly type it?

**Review by kelsos — APPROVED** [review]
lgtm

**Pull request merged**